ui/app/utils/clickhouse/common.ts

```diff
@@ -154,7 +154,10 @@ export function getInferenceTableName(
       return InferenceTableName.JSON;
   }
 }
-
+export const TableBoundsSchema = z.object({
+  first_id: z.string().optional(),
+  last_id: z.string().optional(),
+});
 export interface TableBounds {
   first_id?: string; // UUIDv7 string
   last_id?: string; // UUIDv7 string
```

Comment:
yeah go ahead and derive this type from Zod. Also you should beable to do `.uuid()` on the string too for additional validation.